src/components/Participant/Participant.js

```diff
@@ -30,15 +34,54 @@ function unsetVideo(videoRef) {
 }
 
 function toggleFocus(id, focus) {
-  return focus === 'user' ? participantsActions.unsetFocus() : participantsActions.setFocus(id);
+  return isFocused(focus) ? participantsActions.unsetFocus() : participantsActions.setFocus(id);
 }
 
-function Participant({ id, display, isPublisher, isLocalScreen, streamReady, focus, video }) {
+function renderVideo(id, ref, isPublisher, isLocalScreen, focus, dispatchFn) {
+  return (
+    <video
+      data-testid="participant-video"
+      autoPlay
+      ref={ref}
+      muted={isPublisher}
+      className={isPublisher && !isLocalScreen ? 'mirrored' : ''}
+      onClick={() => dispatchFn(toggleFocus(id, focus))}
+    />
+  );
+}
+
+function renderImage(id, focus, picture, dispatchFn) {
+  const placeholder = '/placeholder.png';
+  const img = isFocused(focus) ? placeholder : picture || placeholder;
+
+  return (
+    <img
+      data-testid="participant-thumbnail"
+      src={img}
+      onClick={() => dispatchFn(toggleFocus(id, focus))}
+    />
+  );
+}
+
+function Participant({
```

Comment:
This component should be simplified (e.g., extracting the video logic to its own component, since it's pretty the same logic used for the speaker), but not in the scope of this PR